Resources/Prototypes/Entities/Objects/Misc/subdermal_implants.yml

```diff
@@ -120,6 +120,11 @@
     - type: Storage
       grid:
       - 0,0,2,2
+      blacklist:
```

Comment:
Pretty sure your also going to have to blacklist `SecretStash` as well (Plushies)!